File: problem/Two_Sneaky_Numbers/getSneakyNumbers.py
class Solution:
    def getSneakyNumbers(self, nums):
        n = len(nums)
        seen = [False] * n
        res = []
        for x in nums:
            if x < n and seen[x]:
                res.append(x)
            elif x < n:
                seen[x] = True
        return res

File: problem/Two_Sneaky_Numbers/getSneakyNumbers.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getSneakyNumbers = function(nums) {
    const n = nums.length;
    const seen = new Array(n).fill(false);
    const res = [];
    for (const x of nums) {
        if (x < n && seen[x]) {
            res.push(x);
        } else if (x < n) {
            seen[x] = true;
        }
    }
    return res;
};